src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,78 @@
+.stop-watch {
+  display: block;
+
+  padding: 50px;
+
+  background: #eae7dc;
+
+  &__box {
+    position: relative;
+
+    margin: auto;
+
+    width: $stop-watch-box;
+    height: $stop-watch-box;
+
+    background-color: #eae7dc;
+    border: 15px dotted #e85a4f;
+    border-radius: 50%;
+  }
+
+  &__center {
+    box-sizing: border-box;
+
+    position: absolute;
+    z-index: 2;
+    top: shift($stop-watch-circle);
+    right: shift($stop-watch-circle);
+
+    width: $stop-watch-circle;
+    height: $stop-watch-circle;
+
+    background-color: #e98074;
+    border: 5px solid #e85a4f;
+    border-radius: 50%;
+  }
+
+  &__minute-arrow {
+    position: absolute;
+    bottom: shift($stop-watch-minute);
+    right: shift($stop-watch-minute);
+
+    width: $stop-watch-minute;
+    height: 225px;
+
+    background-color: #8e8d8a;
+    border-radius: 80%;
+
+    animation-name: time;
+    animation-duration: 60s * 60;
+  }
```

Comment:
I think it will be better to add an animation-timing-function with the value "linear". It will make a second arrow to move such as real second arrow